corehq/apps/domain/forms.py

```diff
@@ -466,6 +467,8 @@ def clean_default_timezone(self):
 
     def save(self, request, domain):
         try:
+            domain.hr_name = self.cleaned_data['hr_name']
+
```

Comment:
hmm, that is rather terrible. I'm open to suggestions on cleaning - would it be better to not wrap it in a try and just surface any errors?